src/main/java/seedu/teachstack/logic/parser/AddressBookParser.java

```diff
@@ -88,6 +89,8 @@ public Command parseCommand(String userInput) throws ParseException {
         case ViewCommand.COMMAND_WORD:
             return new ViewCommandParser().parse(arguments);
 
+        case SummaryCommand.COMMAND_WORD:
```

Comment:
Maybe you can consider to add a parseCommand_summary() test in AddressBookParserTest to pass the codecov.

Reply:
I think it can be something like this
```
@Test
public void parseCommand_summary() throws Exception {
    assertTrue(parser.parseCommand(SummaryCommand.COMMAND_WORD) instanceof SummaryCommand);
    assertTrue(parser.parseCommand(SummaryCommand.COMMAND_WORD + " 3") instanceof SummaryCommand);
}   
```

Reply:
Thanks for the suggestion